### ISSUE TITLE
Keyboard focus issue in some/all Eelium SDL2 tests under Xming/WSL

### ISSUE THREAD

**Issue by olofson** [olofson/eel#117]
(Possibly SDL2-only. If so, it only applies after #71 is merged into master.)

Most/all Eelium tests should quit when hitting ESC, but some of them just ignore that after some unknown series of events.

It looks like it might be an input focus problem between Xming and the WSL console, as the "ignored" ESC escape codes are printed in the console window that the test was launched from, despite the Xming application window looking like it's focused.

The strange thing is that this situation generally occurs after interacting with the window via mouse clicks and motions, which one would think should definitely get the window to take input focus.
